### ISSUE TITLE
change doi api to POST

### ISSUE THREAD

**Issue by codersquid** [researchcompendia/researchcompendia#46]
run the anti-clickjack middleware to prevent use of the service from external requests. Switch from GET to POST to trigger the csrf / clickjack stuff easily

**Comment by codersquid** [comment]
We could further restrict things by requiring a user to be logged in, but I like the idea of someone being able to create new pages without first being logged in.

**Comment by victoriastodden** [comment]
It's ok with me - but probably a good idea to make it easy to change this.
What checks do we have on a new compendium page being legitimate? A
verification of the link to the paper?

Are we restricting one compendium page per paper? I think that is a good
idea, but means internal cross checking and the ability for one compendium
page to manage versions of the paper, not just versions of software or data.

Victoria

**Comment by codersquid** [comment]
For those we have some issues open.
- memberless creation, #16 
- versioning, #11
- page that already exists, #31 

I thought I had something about handling duplicates... I'm not sure what we'd want to do there. I was thinking of having pages that are likely dupes show up when someone starts creating a new doc, and we'd have a page to ask if those pages count, whether they want to claim them, claim collobatorship (we have collaborators that someone can add and linking htem to a user of the site is optional).

anyway... back to this issue. I created it as a reminder to change the service api for the doi information that I created for issue #43. I don't want to lose track of this change I need to make, but it's not my WIP right now.
